web/components/editor.tsx

```diff
@@ -56,6 +63,71 @@ export function useTextEditor(props: {
           class: clsx('no-underline !text-indigo-700', linkClass),
         },
       }),
+      Mention.configure({
```

Comment:
Could we link to docs for tiptap somewhere in this file? Whatever would be helpful.